### ISSUE TITLE
Improve scaling of coin nodes

### ISSUE THREAD

**Issue by jbphet** [phetsims/quantum-measurement#7]
I'm about to commit the first implementation of the coin nodes, and noticed that the letters are scaling differently than the node as a whole.  I'll want to improve this at some point.

**Comment by jbphet** [comment]
I've been asked to focus on getting "Mean: Share and Balance" out the door, and this sim - "Quantum Measurement" - is now on the back burner for a while. I'm going to defer this issue until this sim becomes a priority again.

**Comment by jbphet** [comment]
@AgustinVallejo just discussed this, and we want to make sure we cover both the large coins and the smaller coins.  Specifically, we'd like the letters on the coin to look more consistently sized here:

![image](https://github.com/user-attachments/assets/7a0cd5e6-ec09-472f-8502-cbfa356dc2e9)

...and we'd also like the arrows to look better (more centered vertically) here:

![image](https://github.com/user-attachments/assets/7c609389-6da5-4655-94e0-4dbdeeb002ea)

**Comment by jbphet** [comment]
I've added what I believe is an improved way to scale the content node more consistently.  I tried a couple of things to make it entirely algorithmic, but none of them worked very well given the various shapes of nodes that we were putting on the coin face, so I added a configuration option where a margin could be specified as a proportion of the radius.  I used a custom margin for one of the icons, used the default for all others.  It looks pretty good.

I think that should do it for this one, closing.
